go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/open-ibc/vib-core-smart-contracts/bindings
+module github.com/polymerdao/open-ibc/vibc-core-smart-contracts/v2
```

Comment:
Shouldn't this be `github.com/open-ibc/vibc-core-smart-contracts/xxx`?
Also, `v2` should be `go-bindings` or similar. It's ultimately the package name and should not include versioning, you have the package version for that :)

Reply:
Yeah so the v2 is needed once the major version >= 2.x.x according to the go documentation for modules [1]. With regards to moving the bindings to language specific bindings I don't mind - currently ts-bindings will come from the npm package I think and live in the `src` directory so it'd be nice to move this out into its own directory but I am unsure of its need at this moment. I'd like them all to live in the bindings directory something like `bindings/{go,ts,py}` for example.

[1]] https://github.com/open-ibc/vibc-core-smart-contracts/pull/131#discussion_r1659149330